modules/Microsoft.EventGrid/domains/.test/common/deploy.test.bicep

```diff
@@ -101,5 +101,8 @@ module testDeployment '../../deploy.bicep' = {
       Environment: 'Non-Prod'
       Role: 'DeploymentValidation'
     }
+    topics: [
```

Comment:
I see you also added the topics as a child to the event grid module which is great. I'll update the description accordingly. 

Please just also provide a workflow badge for this module then as well :)

Reply:
Added the module badge also in the PR